### PR TITLE
docs: Update comoshub for v10.0.0 release

### PR DESCRIPTION
**Pull request by mmulji-ic**
### Not to be merged until CosmosHub block height 15213800 has passed

### CODE REVIEW

File: cosmoshub/chain.json [hunk 1]
```diff
@@ -33,41 +33,41 @@
...
     },
     "genesis": {
       "genesis_url": "https://github.com/cosmos/mainnet/raw/master/genesis/genesis.cosmoshub-4.json.gz"
     },
     "versions": [
       {
-        "name": "v9-Lambda",
-        "tag": "v9.1.1",
-        "recommended_version": "v9.1.1",
+        "name": "v10-Epsilon",
```
Comment (dylanschultzie) [inline cosmoshub/chain.json:52]: Please create a new section here, not override v9

Comment (dylanschultzie) [inline cosmoshub/chain.json:52]: Version should be `v10`: https://www.mintscan.io/cosmos/proposals/798

File: cosmoshub/chain.json [hunk 2]
```diff
@@ -33,41 +33,41 @@
...
-        "cosmos_sdk_version": "v0.45.15-ics",
-        "ibc_go_version": "v4.2.1",
+        "cosmos_sdk_version": "v0.45.16-ics",
+        "ibc_go_version": "v4.4.0",
         "consensus": {
           "type": "cometbft",
-          "version": "v0.34.27"
+          "version": "v0.34.28"
         },
         "height": 15213800,
```
Comment (dylanschultzie) [inline cosmoshub/chain.json:64]: Make sure to set the appropriate height and "proposal": 798

**Review by dylanschultzie — CHANGES_REQUESTED** [review]

**Comment by dylanschultzie** [comment]
est time: `Wed Jun 21 2023 20:48:27Z`

**Review by MSalopek — APPROVED** [review]

File: cosmoshub/chain.json [hunk 3]
```diff
@@ -33,21 +33,43 @@
...
+        "binaries": {
+          "linux/amd64": "https://github.com/cosmos/gaia/releases/download/v10.0.1/gaiad-v10.0.1-linux-amd64?checksum=sha256:fcb8210308223d78bc36f3d4c89e2578dcf784994c052cea97efd61f1672cf72",
+          "linux/arm64": "https://github.com/cosmos/gaia/releases/download/v10.0.1/gaiad-v10.0.1-linux-arm64?checksum=sha256:db9b69cf224b410c669fa4f820192890357534e74d4693a744ef915028567462",
+          "darwin/amd64": "https://github.com/cosmos/gaia/releases/download/v10.0.1/gaiad-v10.0.1-darwin-amd64?checksum=sha256:d0bee3b4b243fe1f88ad3258f4648de3a73787434702bcac6e31ca38f81a283a",
+          "darwin/arm64": "https://github.com/cosmos/gaia/releases/download/v10.0.1/gaiad-v10.0.1-darwin-arm64?checksum=sha256:c8124d66ffa99b51da274656f6c3401b1ec9e165a76f3f01699761672e83a136gaiad-v10.0.1-linux-amd64",
+          "windows/amd64": "https://github.com/cosmos/gaia/releases/download/v10.0.1/gaiad-v10.0.1-windows-amd64.exe?checksum=sha256:c02ab2b8fc347f858db1c33fcacafa2467ca550ed83178aee67331762e876926"
+        }
+      },
       {
         "name": "v9-Lambda",
```
Comment (MSalopek) [inline cosmoshub/chain.json:74]: Is this correct?

Comment (MSalopek) [reply cosmoshub/chain.json:74]: Yes it is. It is correct :D

**Review by yaruwangway — APPROVED** [review]

**Review by dylanschultzie — APPROVED** [review]

**Pull request merged**